httpd/handler.go

```diff
@@ -187,6 +194,11 @@ func (h *Handler) serveWrite(w http.ResponseWriter, r *http.Request, user *influ
 		return
 	}
 
+	if h.WriteTrace {
+		b, _ := json.Marshal(bp)
+		h.Logger.Printf("write received by handler: %s", string(b))
```

Comment:
@pauldix and I just spoke about this, it's not quite trivial since `r.Body` is not a simple object. Still, I'll revisit this and see what we can do.